<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4PCJVBQ" ignored="1" name="Test(failed)" tests="4" failures="1" timestamp="2020-11-30T00:20:16 IST" time="0.149" errors="0">
  <testcase name="test2" time="0.026" classname="listeners.LoginTest">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.003" classname="listeners.LoginTest">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.003" classname="listeners.LoginTest">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.004" classname="listeners.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [Rahul1] but found [Rahul]">
      <![CDATA[java.lang.AssertionError: expected [Rahul1] but found [Rahul]
at listeners.LoginTest.test2(LoginTest.java:19)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.0" classname="listeners.LoginTest">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Test(failed) -->
